Dockerfile

```diff
@@ -1,5 +1,5 @@
 # I build the pickup worker that subscribes to the SQS Queue.
-FROM node:16-alpine
+FROM --platform=linux/amd64 node:16-alpine
```

Comment:
The real fix here would be to teach aws-cdk to let us specify the platform build arg when it runs the build but that is  a harder path.